homeassistant/brands/yale.json

```diff
@@ -0,0 +1,5 @@
+{
+  "domain": "yale",
+  "name": "Yale",
+  "integrations": ["august", "yale_smart_alarm", "yalexs_ble"]
```

Comment:
This one overlaps with the August brand.

Reply:
Yes, some Yale locks work with the August integration.
See comment above, it has a similar origin.